src/types/steps.ts

```diff
@@ -47,6 +49,8 @@ export type StepOptionWelcome = {
   nextButton?: string
 }
 
+export type StepOptionAuth = { retries?: number }
```

Comment:
Is `retries` here referring to number of capture retries for the Auth step?

Reply:
Yes.